src/Employer/Employer.Web/Orchestrators/VacanciesOrchestrator.cs

```diff
@@ -154,13 +152,17 @@ private FilteringOptions SanitizeFilter(string filter)
 
         private string GetFilterHeading(int totalVacancies, FilteringOptions filteringOption, string searchTerm)
         {
+            string vacancyStatusPrefix;
             var filterText = filteringOption == FilteringOptions.All ? string.Empty : $" {filteringOption.GetDisplayName().ToLowerInvariant()}";
-            var vacancyText = filteringOption == FilteringOptions.ClosingSoon || filteringOption == FilteringOptions.ClosingSoonWithNoApplications ?
+            var vacancyText = filteringOption == FilteringOptions.Live || filteringOption == FilteringOptions.ClosingSoon || filteringOption == FilteringOptions.ClosingSoonWithNoApplications ?
```

Comment:
This is getting bit hard to read, can the condition be converted into an extension or function
`filteringOption.IsInLiveVacancyOptions() ? "live vacancy" : "vacancy"`
I have removed leading space as ToQuantity can work it out.